systemd/renderd.service

```
@@ -0,0 +1,10 @@
+[Unit]
+Description=Daemon that renders map tiles using mapnik
+Documentation=man:renderd
+After=network.target auditd.service
+
+[Service]
+ExecStart=/usr/bin/renderd -f
```

Shouldn't it be `/usr/local/bin/renderd` ?
That seems to be the default location of `renderd` on my system.

```suggestion
ExecStart=/usr/local/bin/renderd -f
```

---

Presumably you've installed with `make install`? User built programs should be in `/usr/local/bin`, package manager managed programs should be in `/usr/bin`.